pkg/awsiamauth/template.go

```diff
@@ -46,6 +46,7 @@ func (t *TemplateBuilder) GenerateManifest(clusterSpec *cluster.Spec, clusterID
 		"backendMode":        strings.Join(clusterSpec.AWSIamConfig.Spec.BackendMode, ","),
 		"partition":          clusterSpec.AWSIamConfig.Spec.Partition,
 		"kubeVersion124":     clusterSpec.Cluster.Spec.KubernetesVersion == v1alpha1.Kube124,
+		"kubeVersion125":     clusterSpec.Cluster.Spec.KubernetesVersion == v1alpha1.Kube125,
```

Comment:
Maybe we can use a Kube version comparison to check if the version is greater than or equal to 1.24. It might not be the best idea to keep adding versions here for the check.

Reply:
Can we just pass the label as a value and run the logic here?

Reply:
Yes, incorporated both suggestions!